### PR TITLE
support for outlets

### PR DESCRIPTION
**Pull request by paavoleinonen**
Tested against TRÅDFRI Wireless control outlets

### CODE REVIEW

File: src/dirigera/devices/outlet.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    startup_on_off: Optional[StartupEnum]
+
+    def refresh(self) -> None:
+        data = self.dirigera_client.get(route=f"/devices/{self.device_id}")
+        attributes: Dict[str, Any] = data["attributes"]
+        self.device_id = data["id"]
+        self.is_reachable = data["isReachable"]
+        self.custom_name = attributes["customName"]
+        self.is_on = attributes["isOn"]
+        self.startup_on_off = StartupEnum(attributes.get("startupOnOff"))
```
Comment (Leggin) [inline src/dirigera/devices/outlet.py:27]: If startup_on_off is truly optional then make it similar to the changes you made in light.py
```python
        if "startupOnOff" in attributes:
            startup_on_off = StartupEnum(attributes.get("startupOnOff"))
        else:
            startup_on_off = None
```

File: src/dirigera/devices/outlet.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+def dict_to_outlet(data: Dict[str, Any], dirigera_client: AbstractSmartHomeHub):
+    attributes: Dict[str, Any] = data["attributes"]
+
+    return Outlet(
+        dirigera_client=dirigera_client,
+        device_id=data["id"],
+        is_reachable=data["isReachable"],
+        custom_name=attributes["customName"],
+        is_on=attributes["isOn"],
+        startup_on_off=StartupEnum(attributes.get("startupOnOff")),
```
Comment (Leggin) [inline src/dirigera/devices/outlet.py:71]: same as above

**Review by Leggin — COMMENTED** [review]
only minor comments, rest looks great! thank you 🚀

File: src/dirigera/devices/outlet.py (hunk 1 above)
Comment (paavoleinonen) [reply src/dirigera/devices/outlet.py:27]: Done. This optionality is purely based on [Pull request 2](https://github.com/Leggin/dirigera/pull/2). I don't have access to any 3rd part devices to verify.

**Pull request merged**